client-app/src/pages/account/store/modules/payments-list/index.ts

```diff
@@ -0,0 +1,47 @@
+import { Module } from "vuex";
+import { defaultPageSize, paymentsGridFields, paymentsStatuses } from '@common/constants';
+import { localizeGridColumns } from '@common/services/localization.helper.service';
+import { RootState } from "../../types";
+import { actions } from "./actions";
+import { getters } from "./getters";
+import { mutations } from "./mutations";
+import { PaymentsListState } from "./types";
+
+
+// initial state
+export const initialState: PaymentsListState = {
+  errors: {},
+  paymentsList: {
+    listConfig: {
+      columns: paymentsGridFields,
+      pageNumber: 1,
```

Comment:
the same is described here https://github.com/VirtoCommerce/vc-odt-mpa-theme/pull/58